src/main/resources/reference.conf

```diff
@@ -790,6 +800,7 @@ resourceTypes = {
       }
       connect = {
         description = "connect to the Jupyter notebook running on the notebook cluster"
+        authDomainConstrainable = true
```

Comment:
I am a bit confused about how granular the authDomainConstrainable is here. I thought that every resources / actions would inherit that configuration from the workspace after Doug's PR

Reply:
My understanding was you still had to annotate which actions on the child are authDomainConstrainable (like `connect`). @dvoet can you confirm?

Reply:
@rtitle is correct. The auth domain is inherited but which actions have the auth domain applied is controlled by the `authDomainConstrainable` setting.